io-ballerina/csv_stream.bal

```diff
@@ -13,10 +13,9 @@
 // KIND, either express or implied.  See the License for the
```

Comment:
```suggestion
// KIND, either express or implied.  See the License for the
```
The format of how this is written isn't correct. Should be: "For the specific language governing permissions and limitations under the License, see the [License]." Also, shouldn't we include the link to the license?

Reply:
This is the standard license template we have been using.